testsuite/openshift/objects/auth_config/sections.py

```diff
@@ -197,27 +197,41 @@ def add_uma(self, name, endpoint, credentials_secret, **common_features):
 
 
 class ResponseSection(Section):
-    """Section which contains response configuration. todo envoyDynamicMetadata"""
+    """Section which contains response configuration."""
 
     @property
-    def success_headers(self):
-        """Nested dict for most of the section."""
-        return self.section.setdefault("success", {}).setdefault("headers", {})
+    def success(self):
+        """Nested dict for success dictionary."""
+        return self.section.setdefault("success", {})
 
     @modify
-    def clear_success_headers(self):
-        """Clears content of a success headers"""
+    def clear_success(self):
+        """Clears content of a success dictionary."""
         self.success_headers.clear()
 
+    @property
+    def success_headers(self):
+        """Nested dict for items wrapped as HTTP headers."""
+        return self.success.setdefault("headers", {})
+
+    @property
+    def success_dynamic_metadata(self):
```

Comment:
The structure of this is confusing, based on this, I have no idea how the actual object will look. Wouldnt it be better to refactor this entire section to better reflect current state of the CR?

Reply:
If you refactor this section, most likely you will have to make changes to all tests using the request section.

Reply:
It is better now?

Reply:
Its fine for now, we will do further refactor in another PR